src/Microsoft.AspNetCore.Blazor/Json/CamelCase.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Blazor.Json
+{
+    internal static class CamelCase
+    {
+        public static string ToCamelCase(this string value)
```

Comment:
Can we make this not an extension method? It's used in like 4 places 😆

Reply:
OK, fair point. Will do.

Reply:
Done